sdcm/remote.py

```diff
@@ -963,3 +971,429 @@ def run_output_check(self, cmd, timeout=None, ignore_status=False,
 
         if not ignore_status and result.exit_status > 0:
             raise process.CmdError(command=result.command, result=result)
+
+
+class RemoteFabric(object):
+
+    def __init__(self, hostname, user="root", port=22, password="",
+                 key_file=None, wait_key_installed=0, extra_ssh_options=""):
+
+        self.hostname = hostname
+        self.key_file = key_file
+        self.user = user
+        self.port = port
+        self.password = password
+        self.extra_ssh_options = extra_ssh_options
+        logger = logging.getLogger('avocado.test')
+        self.log = SDCMAdapter(logger, extra={'prefix': str(self)})
+
+        self.connect_timeout = 60
```

Comment:
Can we make the connect_timeout configurable?

Reply:
This parameter usually passed upon call the remoter.run method, and not upon the initialization. Therefore, it set in constructor with explicit value. Change constructor to allow configure the connect_timeout on init state.